perfkitbenchmarker/benchmark_sets.py

```diff
@@ -111,7 +111,7 @@
     },
     'mit_set': {
         MESSAGE: 'Massachusetts Institute of Technology benchmark set.',
-        BENCHMARK_LIST: [STANDARD_SET]
+        BENCHMARK_LIST: ['silo']
```

Comment:
Just double checking me mean to drop all other workloads from the MIT set.  You can include the standard_set and SILO like this:

BENCHMARK_LIST: [STANDARD_SET, 'silo']

For can also run the benchmarks like --benchmarks=silo an/or --benchmarks=mit_set